### DISCUSSION TITLE
The things I noticed while using Min.

### DISCUSSION THREAD

**Discussion by mikedegeofroy** [minbrowser/min#1938] [General]
# Hey there! 👋 
So I saw Min on a youtube video, and immediately fell in love with the clean slick design of the browser. But here are the drawbacks and little flaws I noticed. 

My System: 
MacBook Pro (13-inch, 2020, Two Thunderbolt 3 ports)
1,4 GHz Quad-Core Intel Core i5

---
- Importing bookmarks was unnecessary complicated (not very important, but still annoying) 
- I could not import my passwords from a chromium based browser (Brave)
- The touch-bar controls look quite bad (low res), and honestly since nobody really uses them, maybe you're better off without them (or toggle in settings)...
- Why is the dev console not standard command+alt+i **and** f12?
- Not being able to reorganise tabs with the mouse was very unintuitive, but it's kind of a feature as-well... 
- Double clicking on the application window to "zoom" (make it fit the screen) was very annoying...
- I had a hard time setting it as my default browser (found a solution on a thread using `defaultbrowser`, probably not very intuitive for users)
- **Once you reopen a closed tab, you can't navigate backwards, this is a feature I really miss.** 
- The logo could be cleaner?...
---

Overall I love the project, this is all I had to say.

**Comment by PalmerAL** [comment]
Thanks, those are all good points.

I've fixed 3 in 227caa70d21a380e1cf0fe9d9d484485e76868ae; before/after comparison:
<img width="1085" alt="Touch Bar Shot 2022-03-10 at 8 42 53 PM" src="https://user-images.githubusercontent.com/10314059/157798959-ad5aabbd-a712-4eda-8de1-d9e04a52503f.png">
<img width="1085" alt="Touch Bar Shot 2022-03-10 at 8 42 55 PM" src="https://user-images.githubusercontent.com/10314059/157798967-b450b97e-339c-4309-acc2-11ad796e5ae3.png">

I think ede0d413492e32133d59cfc5b0b6cd71108f23fa should fix 7, needs more testing though.


Better import from other browsers would be nice, but is a fairly large project; I don't anticipate working on it anytime soon.

The issues with tab dragging and double-clicking to maximize are a consequence of filling the entire titlebar with tabs. In order to fix this, I think we would have to do what other browsers do and have a separate draggable area, but that would mean either making the tabs smaller or the titlebar taller, which I think isn't ideal.

**Reply by mikedegeofroy** [reply]
Wow! That looks great! Really stoked for updates on the project! :) 

About the titlebar, yes, that's exactly what I thought, and it's not worth sacrificing the design. Maybe separately recognising double clicks (for maximising and minimising) would be possible, but it's far from being a priority imho.
